sample/client.http

```diff
@@ -0,0 +1,14 @@
+POST http://localhost:8080/sampleservice/bans HTTP/1.1
```

Comment:
hi @gremlinflat what is this file is used for? If it's only for personal testing purpose, don't forget to remove it. Thanks